components/text/dwdiff/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Matthew R. Trower
+#
+
+BUILD_BITS=64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=         dwdiff
+COMPONENT_VERSION=      2.1.4
+COMPONENT_SUMMARY=      dwdiff - a diff program that operates at the word level
+COMPONENT_PROJECT_URL=  https://os.ghalkes.nl/dwdiff.html
+COMPONENT_FMRI=         text/dwdiff
+COMPONENT_CLASSIFICATION=System/Text Tools
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.bz2
+COMPONENT_ARCHIVE_URL=  https://os.ghalkes.nl/dist/$(COMPONENT_ARCHIVE)
+COMPONENT_SIG_URL=      $(COMPONENT_ARCHIVE_URL).sig
+COMPONENT_ARCHIVE_HASH= \
+	sha256:df16fec44dcb467d65a4246a43628f93741996c1773e930b90c6dde22dd58e0a
+COMPONENT_LICENSE=      GPLv3
+
+TEST_TARGET=$(NO_TESTS)
+
+include $(WS_MAKE_RULES)/common.mk
+
+CONFIGURE_OPTIONS+=	INSTALL=/usr/ucb/install
+CONFIGURE_OPTIONS+=	CFLAGS=-D_POSIX_PTHREAD_SEMANTICS
+
+# Wants to be built in-tree, so copy source tree to build dir
+CONFIGURE_SCRIPT = $(@D)/configure
```

Comment:
Is this really needed?

Reply:
dwdiff isn't designed with out-of-tree builds in mind.  It is full of relative references to in-tree files.  It might build without this option, but then it's not an in-tree build --- configure will be simultaneously referencing components from both the build directory (containing a full copy of the source) and the original source directory.

With this option, we are just doing a standard in-tree build, which is semantically what I am trying to achieve in the first place.   If you really want me to remove it though, I can try to do so.

Reply:
Okay, makes sense.